session.py

```diff
@@ -245,6 +258,48 @@ def post_mealitem(self, nutritionplanId, mealId, ingredientId, amount):
         response_post_mealitem = self.s.post(post_mealitemURL, json = payload_mealitem , headers=mealitem_post_headers )
         return [response_post_mealitem , mySession.check(response_post_mealitem)]
     
+    def delete_workout(self, workoutId):
```

Comment:
The 2 delete methods have a pretty similar structure. You know where I'm going with this :smile: